### PR TITLE
Expose pool_whatcontainsdep() in the bindings

### LINKED ISSUE #335: Expose and document pool_whatcontainsdep() in the bindings

### PR DESCRIPTION
**Pull request by dralley**
Document both pool_whatcontainsdep() and pool_whatmatchesdep()

closes #335

### CODE REVIEW

**Comment by ignatenkobrain** [comment]
LGTM

File: doc/libsolv-bindings.txt [hunk 1]
```diff
@@ -495,6 +495,20 @@ a Dep object or a simple Id as argument.
...
+	solvables = pool.whatcontainsdep(keyname, dep)
+
+Return all solvables for which keyname contains the dependency.
+
+	Solvable *whatmatchesdep(Id keyname, DepId dep, Id marker = -1)
+	my @solvables = $pool->whatmatchesdep($keyname, $sdep)
+	solvables = pool.whatmatchesdep(keyname, dep)
+	solvables = pool.whatmatchesdep(keyname, dep)
+
+Return all solvables that have dependencies in keyname that match the dependency.
```
Comment (dralley) [inline doc/libsolv-bindings.txt]: I'm not sure I totally understand the distinction between these two functions, so these descriptions might be incorrect or inadequate.

**Comment by mlschroe** [comment]
The difference is that whatcontainsdep is an "exact" match, so `A > 3` will only be matched by a package that contains a `A > 3` dependency. Whereas whatmatchesdep does a real match, i.e. `A > 3` will be matched by a package that contains `A = 4`.

**Comment by dralley** [comment]
Thanks @mlschroe , I suppose the descriptions are OK then.

**Pull request merged**